### PR TITLE
Bump ps2census v0.4.2

### LINKED ISSUE #117: Bump ps2census to v0.4.2
Solves #106, #114

### PR DESCRIPTION
**Pull request by microwavekonijn**
Closes #117, #106, #114.

Before this gets merged. Some things that I want to ask:
- First errors in the handlers will now crash the application as a result of a change in the ps2census lib. Should we let it just crash or do we want to add a helper to catch the errors?;
- Shall I include a redis cache according to the specs of the ps2census contract?;
- ~~Should I add a timeout to the kernel on termination(so that discord is able to log)?~~

### CODE REVIEW

**Comment by Maelstromeous** [comment]
> First errors in the handlers will now crash the application as a result of a change in the ps2census lib. Should we let it just crash or do we want to add a helper to catch the errors?;

Application exceptions should crash the app. If we're happy with absorbing an error it should be a logged error as normal, but things which should not happen need to crash (and log) so we can properly investigate them.

> Shall I include a redis cache according to the specs of the ps2census contract?;

Please, feel free to update my PR, apologies I've been busy as of late

File: src/handlers/aggregate/global/GlobalFactionCombatAggregate.ts [hunk 1]
```diff
@@ -4,9 +4,13 @@ import {getLogger} from '../../../logger';
 import {inject, injectable} from 'inversify';
 import MongooseModelFactory from '../../../factories/MongooseModelFactory';
 import {TYPES} from '../../../constants/types';
-import {GlobalFactionCombatAggregateSchemaInterface, GlobalFactionCombatAggregateSubSchemaInterface} from '../../../models/aggregate/global/GlobalFactionCombatAggregateModel';
+import {
```
Comment (Maelstromeous) [inline src/handlers/aggregate/global/GlobalFactionCombatAggregate.ts:7]: Formatting changed

File: src/handlers/census/events/DeathEvent.ts [hunk 2]
```diff
@@ -116,10 +115,6 @@ export default class DeathEvent {
 
         this.isHeadshot = event.is_headshot;
 
-        // Additional logic to figure out things Census doesn't tell us...
-        this.isSuicide = this.characterId === this.attackerCharacterId;
-
-        // TK logic, does NOT include NSO. If same faction but not same character
-        this.isTeamkill = this.attackerFaction === this.characterFaction && !this.isSuicide;
+        this.killType = event.kill_type;
```
Comment (Maelstromeous) [inline src/handlers/census/events/DeathEvent.ts:118]: Have you rigourlessly tested this? Wouldn't want a repeat of what we had before where every kill was a headshot :-/

Comment (microwavekonijn) [reply src/handlers/census/events/DeathEvent.ts:118]: This is a computed property not a casted. So basically a function which will return a result but as a getter.

**Pull request merged**